packages/components/form/src/form-item.vue

```diff
@@ -135,7 +137,11 @@ const formItemClasses = computed(() => [
   formContext?.requireAsteriskPosition === 'right'
     ? 'asterisk-right'
     : 'asterisk-left',
-  { [ns.m('feedback')]: formContext?.statusIcon },
+  {
+    [ns.m('feedback')]: formContext?.statusIcon,
+    [ns.m(`label-${props?.labelPosition || formContext?.labelPosition}`)]:
+      props.labelPosition || formContext?.labelPosition,
```

Comment:
`props.labelPosition || formContext?.labelPosition` is used in multiple places and can be extracted into a computed value.